game/glutil.cc

```diff
@@ -24,6 +24,8 @@ namespace glutil {
 
 	void VertexArray::clear() {
 		m_vertices.clear();
+                glDeleteVertexArrays(1, &m_vao);
+                glDeleteBuffers(1, &m_vbo);
```

Comment:
Incorrect indentation (spaces, should be tab).

Reply:
This is wrong, VertexArray is a client buffer, VAO and VBO should only be created and destroyed once.

Reply:
'k